logic/scripts/token_stats.py

```diff
@@ -239,11 +246,96 @@ def fetch_ogn_stats(ogn_usd_price,staked_user_count,staked_token_count,ogn_stake
 
     return out_data
 
-def update_circulating_supply(circulating_supply):
+def fetch_ogv_stats(ogv_usd_price):
+    total_supply = 4071219784
```

Comment:
Should this be hard-coded or pulled from elsewhere in this script?

Reply:
this was hardcoded for OGN so just did the same here, can change it later maybe once this goes through